trade-finance-manager-ui/styles/table.scss

```diff
@@ -156,4 +156,26 @@ table.premium-payments-table {
   .payments-list-item {
     list-style-type: none;
   }
-}
\ No newline at end of file
+}
+
+table.utilisation-report-reconciliation-table {
+  table-layout: fixed;
+
+  .ukef-width-one-tenth {
+    width: 10% !important;
+  }
+
+  .ukef-width-one-quarter {
+    width: 25% !important;
+  }
+}
+@media (max-width:65em) {
+  table.utilisation-report-reconciliation-table {
+    width: 55em;
+
+    .govuk-table__header,
+    .govuk-table__cell {
+      white-space: normal;
```

Comment:
We require this to override the `nowrap` value the scrollable pane applies (which causes elements to overlap)